packages/abi-gen/src/utils.ts

```diff
@@ -56,7 +56,7 @@ export const utils = {
                     const componentType = `${component.name}: ${componentValueType}`;
                     return componentType;
                 });
-                const tsType = `{${componentsType}}`;
+                const tsType = `{${componentsType.join(';')}}`;
```

Comment:
If this is a bug fix, mind adding a CHANGELOG entry?

Reply:
It's not a bug fix. Before it was using a comma and that's also OK. I changed it to ';' because the linter was complaining. Users should not be affected so I don't think it belongs to CHANGELOG